google-cloud-bigtable/src/main/resources/changestream.json

```diff
@@ -0,0 +1,1232 @@
+                    "family_name": "family"
+                  }
+                }
+              },
+              {
+                "mutation": {
+                  "delete_from_column" : {
+                    "family_name": "family",
+                    "column_qualifier": "dg==",
+                    "time_range": {
+                      "start_timestamp_micros": 5000,
+                      "end_timestamp_micros": 15000
+                    }
+                  }
+                }
+              },
+              {
+                "mutation": {
+                  "set_cell": {
+                    "family_name": "family",
+                    "column_qualifier": "0000000000000000",
+                    "timestamp_micros": 1000,
+                    "value": "dg=="
+                  }
+                }
+              }
+            ],
+            "done": true
+          }
+        }
+      ],
+      "results": [
+        {
+          "record": {
+            "change_stream_mutation": {
+              "row_key": "0000000000000000",
+              "type": "USER",
+              "source_cluster_id": "source-cluster-id",
+              "commit_timestamp": "2022-07-01T00:00:00Z",
+              "low_watermark": "2022-07-01T00:00:00Z",
+              "tiebreaker": 100,
+              "token": "data-change-token",
+              "mutations": [
+                {
+                  "delete_from_family": {
+                    "family_name": "family"
+                  }
+                },
+                {
+                  "delete_from_column" : {
+                    "family_name": "family",
+                    "column_qualifier": "dg==",
+                    "time_range": {
+                      "start_timestamp_micros": 5000,
+                      "end_timestamp_micros": 15000
+                    }
+                  }
+                },
+                {
+                  "set_cell": {
+                    "family_name": "family",
+                    "column_qualifier": "0000000000000000",
+                    "timestamp_micros": 1000,
+                    "value": "dg=="
+                  }
+                }
+              ]
+            }
+          },
+          "error": false
+        }
+      ]
+    },
+    {
+      "description": "1 incomplete logical mutation",
+      "api_responses": [
+        {
+          "data_change": {
+            "row_key": "0000000000000000",
+            "type": "USER",
+            "source_cluster_id": "source-cluster-id",
+            "commit_timestamp": "2022-07-01T00:00:00Z",
+            "tiebreaker": 100,
+            "token": "data-change-token",
+            "chunks": [
+              {
+                "mutation": {
+                  "delete_from_family": {
+                    "family_name": "family"
+                  }
+                }
+              }
+            ]
+          }
+        }
+      ],
+      "results": [
+        {
+          "error": true
```

Comment:
This is an error because there's no "done": true right?

Reply:
Right! Updated the description.